wordpress-mobile/gutenberg-mobile
# Fix Missing Placeholder on Quote Blocks

Fixes #1970 

### Description
This change is for fixing missing placeholder text on Quote and Pullquote blocks on Android.
See Gutenberg PR for more details.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/22561

### To test:

1. Add Quote and Pullquote blocks, verify that the "Enter Quote" placeholder text is visible.
2. Try adding multiline quotes and merging other blocks into a quote block, make sure no text is lost.
3. Smoke test other rich text based blocks to ensure there is no unintended behavior.

### Screenshot
<img width="446" alt="Screen Shot 2020-05-22 at 6 20 29 PM" src="https://user-images.githubusercontent.com/1103838/82718724-1796c280-9c59-11ea-94bd-787e86a5fb2c.png">

submission checklist:
